sqlitecluster/SQLiteNode.cpp

```diff
@@ -1177,6 +1183,9 @@ void SQLiteNode::_onMESSAGE(Peer* peer, const SData& message) {
                             PWARN("Lower-priority peer is trying to stand up while we are " << stateNames[_state]
                                   << ", but we don't have a majority of the cluster so reconnecting and SEARCHING.");
                             _reconnectAll();
+                            // TODO: This puts us in an ambiguous state if we switch to SEARCHING from MASTERING,
```

Comment:
Maybe the comment would have been clearer if I'd worded it `from MASTERING to SEARCHING`, but it's the opposite order from what you're concerned about.